web/content/docs/benchmarks/hydro-thermal/decovalex-TH.md

```diff
@@ -0,0 +1,131 @@
++++
+project = "Parabolic/HT/StaggeredCoupling/ADecovalexTHMCBasedHTExample/th_decovalex.prj"
+author = "Wenqing Wang"
+date = "2018-02-13T12:42:53+01:00"
+title = "An example about the coupled thermal hydraulic processes in the FEBEX type repository"
+weight = 161
+
+[menu]
+  [menu.benchmarks]
+    parent = "hydro-thermal"
+
++++
+
+{{< data-link >}}
+
+## Problem description
+
+This example is based on the The TASK D_THM1 of the DECOVALEX-THMC project.
+The TASK D_THM1 of the DECOVALEX-THMC project studies the coupled thermal hydraulic
+ and mechanical (THM) processes in the FEBEX type nuclear waster repository.
+ In the FEBEX type repository, nuclear waster canisters are stored in the
+ drifts excavated in the deep rock mass of granite, which is fully water
+ saturated, and they are sealed with bentonite. Initiallly, the betonite,
+ the sealing material, is partially saturated. TASK D_THM1 defines a 2D model
+ about the FEBEX type repository \cite BirEtAl:2008. In this example,
+ TASK D_THM1 is simplified in order to test the staggered scheme for TH process
+ in ogs6. The simplifications are
+<ol>
+<li><p>resizing the domain to an area that can represent the near field of an
+ installed nuclear water canister,</p></li>
+<li><p>assuming the the betonite is fully saturated from the beginning,</p></li>
+<li><p>ignoring the mechnical process.</p></li>
+</ol>
+<p>With such simplifications, the geometry of the present example is illustrated
+ in the following figure:</p>
+<p><img src="decovalex_TH_domain.png" alt="image" /></p>
```

Comment:
Looking at the build web pages it seems the images are missing. https://jenkins.opengeosys.org/job/ufz/job/ogs/job/PR-2076/4/Web/

Reply:
Tried by using the following image including syntax:
```
{{< img src="../decovalex_TH_domain_pT.png" > }}
```

Reply:
It works.